src/main/java/uk/gov/hmcts/ccd/v2/internal/resource/UIWorkbasketInputsResource.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.ccd.v2.internal.resource;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.springframework.hateoas.ResourceSupport;
+import uk.gov.hmcts.ccd.domain.model.search.WorkbasketInput;
+import uk.gov.hmcts.ccd.v2.internal.controller.UIDefinitionController;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class UIWorkbasketInputsResource extends ResourceSupport {
+
+    @JsonUnwrapped
+    private WorkbasketInput[] workbasketInputs;
```

Comment:
Instead of unwrapping it, I think it would be better to duplicate the contract and copy each field across. While it does seem wrong, it achieves the primary goal of that class, which is to act as a definitive contract for the API. I'm concerned that by relying on `WorkBasketInput` for the contract, it could easily be broken by mistake because it may not be clear that `WorkbasketInput` is the contract

Reply:
I think the unwrapping is not needed here and I will remove. Are you suggesting I should copy the `WorkbasketInput` class exactly as is and call it `UIWorkbasketInput` and then have an array of `UIWorkbasketInput` inputs isntead?